pymobiledevice3/cli/cli_common.py

```diff
@@ -57,6 +75,22 @@ def udid(ctx, param, value):
         if '_PYMOBILEDEVICE3_COMPLETE' in os.environ:
             # prevent lockdown connection establishment when in autocomplete mode
             return
+        if value is None:
```

Comment:
check instead `if value is not None` so you can simply return before and prevent the inner nesting

Reply:
but then I need to return `LockdownClient(serial=value)` again if there is less then 2 'USB' devices connected

Reply:
correct, but according to the zen of python:
*"Flat is better than nested."*

In general  we prefer the code avoids nesting. If something can be checked and avoided beforehand - it should

Reply:
ok, changed :)